### PR TITLE
402 Remove STIN and RFPOT !!!MERGE 402_frontend FIRST!!!

### PR DESCRIPTION
**Pull request by sadlerw**
* clean em out

screen shot 2019-01-05 at 2 01 53 pm

### CODE REVIEW

File: shared/src/business/useCases/getCaseTypes.interactor.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+ *
+ * @param userId
+ * @returns {Promise<*>}
+ */
+exports.getCaseTypes = async ({ userId }) => {
+  if (!isAuthorized(userId, PETITION)) {
+    throw new UnauthorizedError('Unauthorized');
+  }
+
+  return Case.getCaseTypes();
```
Comment (dknesek) [inline shared/src/business/useCases/getCaseTypes.interactor.js]: How does the use case interactor guarantee its contract with the controller/presenter? That is, how does the interactor ensure the structure of the result is valid?

File: shared/src/business/useCases/getCaseTypes.interactor.test.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+const { getCaseTypes } = require('./getCaseTypes.interactor');
+
+describe('Get case types', () => {
+  beforeEach(() => {});
+
+  it('returns a collection of case types', async () => {
+    const caseTypes = await getCaseTypes({
+      userId: 'taxpayer',
+    });
+    expect(caseTypes.length).toEqual(12);
```
Comment (dknesek) [inline shared/src/business/useCases/getCaseTypes.interactor.test.js:23]: How do we test that this function really is working as the caller expects?

File: shared/src/business/useCases/getProcedureTypes.interactor.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+ *
+ * @param userId
+ * @returns {Promise<*>}
+ */
+exports.getProcedureTypes = async ({ userId }) => {
+  if (!isAuthorized(userId, PETITION)) {
+    throw new UnauthorizedError('Unauthorized');
+  }
+
+  return Case.getProcedureTypes();
```
Comment (dknesek) [inline shared/src/business/useCases/getProcedureTypes.interactor.js]: How does the use case interactor guarantee its contract with the controller/presenter? That is, how does the interactor ensure the structure of the result is valid?

File: shared/src/business/useCases/getTrialCities.interactor.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+ * @param userId
+ * @param procedureType
+ * @returns {Promise<*>}
+ */
+exports.getTrialCities = async ({ userId, procedureType}) => {
+  if (!isAuthorized(userId, PETITION)) {
+    throw new UnauthorizedError('Unauthorized');
+  }
+
+  return Case.getTrialCities(procedureType);
```
Comment (dknesek) [inline shared/src/business/useCases/getTrialCities.interactor.js]: How does the use case interactor guarantee its contract with the controller/presenter? That is, how does the interactor ensure the structure of the result is valid?

File: shared/src/business/useCases/getTrialCities.interactor.test.js [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+describe('Get trial cities', () => {
+  beforeEach(() => {});
+
+  it('returns a collection of trial cities', async () => {
+    const caseTypes = await getTrialCities({
+      userId: 'taxpayer',
+      procedureType: 'Small',
+    });
+    expect(caseTypes.length).toEqual(74);
```
Comment (dknesek) [inline shared/src/business/useCases/getTrialCities.interactor.test.js]: How do we test that this function really is working as the caller expects?

**Review by dknesek — CHANGES_REQUESTED** [review]

**Review by codyseibert — APPROVED** [review]

**Pull request merged**